crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1361,13 +1370,10 @@ impl TryFrom<NativeSuiSystemStateSummary> for SuiSystemStateSummary {
 impl From<&TransactionKind> for TransactionBlockKind {
     fn from(value: &TransactionKind) -> Self {
         match value {
-            TransactionKind::ConsensusCommitPrologue(x) => {
-                let consensus = ConsensusCommitPrologueTransaction {
-                    epoch_id: x.epoch,
-                    round: Some(x.round),
-                    timestamp: DateTime::from_ms(x.commit_timestamp_ms as i64),
-                };
-                TransactionBlockKind::ConsensusCommitPrologueTransaction(consensus)
+            TransactionKind::AuthenticatorStateUpdate(_) => {
```

Comment:
You should probably rebase as there's been some changes to this file. Rather than unimplemented we render the kinds as strings

Reply:
Rebased.